First time installation error

Unable to install OpenLane in Ubuntu machine due to version detail missing in ciel for pdk. 
Environment :
```
xuxail@xuxail-ubuntu:~$` git --version
python3 --version
docker --version
make --version
git version 2.43.0
Python 3.12.3
Docker version 28.4.0, build d8eb465
GNU Make 4.3
Built for x86_64-pc-linux-gnu
Copyright (C) 1988-2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by `law.
```


The error I am getting:

```
xuxail@xuxail-ubuntu:~/OpenLane$ make
make[1]: Entering directory '/home/xuxail/OpenLane'
ff5509f65b17bfa4068d5336495ab1718987ff69: Pulling from the-openroad-project/openlane
Digest: sha256:d7e2bdd21e757e3f0b15d6ea08c36d9a96653c31b4fb2fafd4de88c9b6a3f3fa
Status: Image is up to date for ghcr.io/the-openroad-project/openlane:ff5509f65b17bfa4068d5336495ab1718987ff69
ghcr.io/the-openroad-project/openlane:ff5509f65b17bfa4068d5336495ab1718987ff69
make[1]: Leaving directory '/home/xuxail/OpenLane'
./venv/bin/ciel enable --pdk sky130A
Usage: ciel enable [OPTIONS] <--pdk-family sky130|gf180mcu|ihp-sg13g2>
                   <VERSION>
Try 'ciel enable --help' for help.

Error: Missing argument '<VERSION>'.
make: *** [Makefile:114: pdk] Error 2
```

--- a/ciel/click_common.py
+++ b/ciel/click_common.py
@@ -44,6 +44,9 @@ class VersionArgument(click.Argument):
 
     def process_value(self, ctx, value):
         try:
+            # handle sentinels in click ≥ 8.3.0
+            if hasattr(value, "name") and value.name == "UNSET":
+                value = None
             tool_metadata_file_path = None
             if "tool_metadata_file_path" in ctx.params:
                 tool_metadata_file_path = ctx.params["tool_metadata_file_path"]

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ciel"
-version = "2.3.0"
+version = "2.3.1"
 description = "An PDK builder/version manager for PDKs in the open_pdks format"
 authors = ["Mohamed Gaber <me@donn.website>", "Efabless Corporation"]
 readme = "Readme.md"

